paraatm/tests/test_weather_parser.py

```diff
@@ -0,0 +1,21 @@
+#!/home/anaconda3 python
+# -*- coding: utf-8 -*-
+
+"""The test function for the weather data parser"""
+
+import os
+from paraatm.io.weather_cube_ET import weather_cube_generator
+
+# set parameters to the function in a dictionary
+cfg = {'cube_size': 20, 'resize_ratio': 1, 'downsample_ratio': 1}
+
+# Get the current directory
+cwd = os.path.dirname(os.getcwd())
+
+# Directory to weather data path
+cfg['weather_path'] = os.path.join(cwd, 'sample_data/weather_parser/echotop_data/')
+cfg['trajectory_path'] = os.path.join(cwd, 'sample_data/weather_parser/Weather_demo_track.csv')
```

Comment:
This won't actually work when your test is called from anywhere other than the `paraatm` directory.  The best way is to get the location of the current module, and then find the `sample_data` directory relative to that.  Please see how this is done in [test_para_atm.py](../blob/master/paraatm/tests/test_para_atm.py).

Reply:
That's a good point. This code assumes the test function stays in the /tests directory. I will change it to your approach.